internal/pkg/externalplugins/label/label.go

```diff
@@ -157,62 +170,67 @@
 	if err != nil {
 		return err
 	}
-	labels, err := gc.GetIssueLabels(org, repo, e.Issue.Number)
+	issueLabels, err := gc.GetIssueLabels(org, repo, e.Issue.Number)
 	if err != nil {
 		return err
 	}
 
-	existingLabels := map[string]string{}
+	repoExistingLabels := map[string]string{}
 	for _, l := range repoLabels {
-		existingLabels[strings.ToLower(l.Name)] = l.Name
+		repoExistingLabels[strings.ToLower(l.Name)] = l.Name
 	}
 
-	excludeLabelsSet := sets.NewString(excludeLabels...)
+	excludeLabelsSet := sets.NewString()
+	for _, l := range excludeLabels {
+		excludeLabelsSet.Insert(strings.ToLower(l))
+	}
 
 	var (
 		nonexistent         []string
+		noSuchLabelsInRepo  []string
 		noSuchLabelsOnIssue []string
 		labelsToAdd         []string
 		labelsToRemove      []string
 	)
 
-	// Get labels to add and labels to remove from regexp matches
+	// Get labels to add and labels to remove from the Regexp matches.
+	// Notice: The returned label is lowercase.
 	labelsToAdd = append(getLabelsFromREMatches(labelMatches),
-		getLabelsFromGenericMatches(customLabelMatches, additionalLabels)...)
+		getLabelsFromGenericMatches(customLabelMatches, additionalLabels, &nonexistent)...)
```

Comment:
I don't think we need to use pointer in this place, so why use it?

Reply:
Because we want to modify this variable in the function body, and this function here is a single return value.

https://github.com/kubernetes/test-infra/blob/e9bafa1cc6be4c8fae4046cef5aeccaa28b3a648/prow/plugins/label/label.go#L175